stallman/producao.md

```diff
@@ -1,4 +1,89 @@
 [Voltar](intro.md)
 
 Produção (livros, artigos, programas de computador, etc)
-====
\ No newline at end of file
+====
+# Livros publicados:
+
+### Free Software, Free Society: Selected Essays of Richard M. Stallman
+<img src="https://images-na.ssl-images-amazon.com/images/I/51fgPvci7CL._SX367_BO1,204,203,200_.jpg" alt="Capa do Livro Free Software, Free Society: Selected Essays of Richard M. Stallman" width="200"/> <br />
+Traduzindo para o português **Software Livre, Sociedade Livre: Ensaios Selecionados de Richard M. Stallman** é uma coleção de escritos de Richard Stallman (principalmente ensaios, com artigos ocasionais, entrevistas e rascunhos de discursos). Apresenta a história e os tópicos de desenvolvimento do Projeto GNU e da *Free Software Foundation*, e explica a posição filosófica do autor sobre o movimento do software livre, abrangendo tópicos como ética do software, direitos autorais e lei de patentes e práticas de negócios na aplicação ao software de computador. Os autores propõem licenças de software livre (principalmente GPL) como solução para os problemas sociais criados pelo software proprietário. [Acessar obra](https://www.gnu.org/philosophy/fsfs/rms-essays.pdf). <br />
```

Comment:
Gostei do link direto para a obra, mas, para ficar padronizado com as outras páginas deste repositório, usem o sistema de nota de rodapé, conforme explicado [aqui](https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes).

Reply:
Adicionado nota de rodapé.